### PR TITLE
bcm43xxx: remove unuse function

### PR DESCRIPTION
**Pull request by zhhyu7**
## Summary
I added a function that I didn't use when I upstream another patch https://github.com/apache/nuttx/pull/10709. Delete it

## Testing
sim:local

### CODE REVIEW

File: tools/configure.sh [hunk 1]
```diff
@@ -270,6 +270,9 @@ if [ -z "${appdir}" ]; then
 
     if [ -d "${TOPDIR}/../apps-${CONFIG_VERSION_STRING}" ]; then
       appdir="../apps-${CONFIG_VERSION_STRING}"
+    else
+        echo "ERROR: Could not find the path to the appdir"
```
Comment (pkarashchenko) [inline tools/configure.sh:274]: I think this is unrelated change

Comment (zhhyu7) [reply tools/configure.sh]: > I think this is unrelated change

yes, I corrected this mistake. thanks

**Review by xiaoxiang781216 — APPROVED** [review]

**Review by xiaoxiang781216 — APPROVED** [review]

**Review by xiaoxiang781216 — APPROVED** [review]
